src/irma-functions.js

```diff
@@ -45,7 +52,7 @@ function irmaDoSession (request) {
         // Front-end options
         language: 'en',
         translations: {
-            header: 'Sign the agreement with <i class="irma-web-logo">IRMA</i>',
+            header: `${header} <i class="irma-web-logo">IRMA</i>`,
```

Comment:
Hum... Maybe we can add a little InternetNZ icon before the message later... Would be cool.
`<i class="inz-logo">INZ</i> ${header} <i class="irma-web-logo">IRMA</i>`

Reply:
Will be awesome :)